### PR TITLE
Fixing python issues with edb-ansible

### PR DESCRIPTION
**Pull request by vibhorkumar123**
@jt-edb 
Can you please validate the changes in this PR? We are planning to release these changes today. This change is also important for the Proof of Technology work.

Thank you!

### CODE REVIEW

File: roles/setup_pem/tasks/setup_pemserver.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  register: output
+
+- name: Remove pem server based on force_pem_install
+  block:
+    - import_tasks: rm_pem_server_install.yml
+  when: force_pem_install
+  become: yes
+  delegate_to: "{{ item.value.public_ip }}"
+
+- name: Install and configure postgres on centos7 or rhel7
```
Comment (jt-edb) [inline roles/setup_pem/tasks/setup_pemserver.yml:24]: Name shouldn't be: **Install and configure PEM server** here ?

File: roles/setup_pem/tasks/main.yml [hunk 2]
```diff
@@ -43,16 +43,30 @@
...
+     - item.value.node_type == 'pemserver'
+     - pem_server_exists
+  with_dict: "{{ servers }}"
+
 - name: Execute setup_pem for all nodes
-  include_tasks: setup_pem.yml
-  when: pem_server_exists
+  include_tasks: setup_pemagent.yml
+  when: 
+     - item.value.node_type != 'pemserver'
```
Comment (jt-edb) [inline roles/setup_pem/tasks/main.yml:68]: No need to setup a PEM agent on PEM server database ?

**Review by jt-edb — APPROVED** [review]
LGTM, only 2 comments/questions

File: roles/setup_pem/tasks/main.yml (hunk 2 above)
Comment (vibhorkumar123) [reply roles/setup_pem/tasks/main.yml:68]: Yes. setup_pemserver automatically installs pemagent for pemserver. Also, pemagent on pemserver is different from the pemagent on other nodes like primary and standby

File: roles/setup_pem/tasks/setup_pemserver.yml (hunk 1 above)
Comment (vibhorkumar123) [reply roles/setup_pem/tasks/setup_pemserver.yml:24]: Yeah. Fixing it.

Comment (vibhorkumar123) [reply roles/setup_pem/tasks/setup_pemserver.yml]: Fixed

Comment (jt-edb) [reply roles/setup_pem/tasks/setup_pemserver.yml]: Thanks

**Pull request merged**